.devcontainer/Dockerfile

```diff
@@ -9,6 +9,18 @@ ARG USER_UID=1000
 ARG USER_GID=$USER_UID
 RUN if [ "$USER_GID" != "1000" ] || [ "$USER_UID" != "1000" ]; then groupmod --gid $USER_GID vscode && usermod --uid $USER_UID --gid $USER_GID vscode; fi
 
+# Add PSQL repo
+RUN sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+RUN apt-get update
+
+# Install the PSQL client
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends postgresql-client
+
+ARG NODE_VERSION="lts/*"
+ARG INSTALL_NODE="true"
+RUN if [ "${INSTALL_NODE}" = "true" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
+
```

Comment:
Me molesta un poco perder la atomicidad de los contenedores, porque claramente esto le agrega tiempo de construcción al contenedor.
Dado que tenemos un docker-compose.yml encuentro que se podría especializar esta tarea de hacer backups en otro contenedor.

Reply:
Respecto a node, también encuentro que puede estar en otro contenedor.

Reply:
El devcontainer solo se instala una vez en el entorno de desarrollo y node es necesario en el entorno principal porque es parte del build and run.

Reply:
Nótese que Node no está actuando como servicio. Simplemente ee utiliza para manejar las dependencias con Webpack, y por ende moverlo a su propio contenedor solo para correr un `npm install` suena innecesario. Por lo demás tendríamos que lidiar con un sistema de archivos compartido, lo que es siempre caótico.

Esta configuración de devcontainer es la recomendada para entornos de Python + PostgreSQL, de hecho el segundo cambio simplemente devuelve la configuración recomendada.